src/azure-cli/azure/cli/command_modules/appservice/static_sites.py

```diff
@@ -120,7 +120,7 @@ def set_staticsite_function_app_settings(cmd, name, setting_pairs, resource_grou
         setting_dict[key] = value
 
     return client.create_or_update_static_site_function_app_settings(
-        resource_group_name, name, kind=None, properties=setting_dict)
+        resource_group_name, name, app_settings=setting_dict)
```

Comment:
I saw same change recently by Jian Song.

Reply:
Ah I see :) if that change is merged I will rebase.